ocs_ci/ocs/ui/deployment_ui.py

```diff
@@ -224,8 +224,14 @@ def install_lso_cluster(self):
             self.do_click(
                 locator=self.dep_loc["all_nodes_create_sc"], enable_screenshot=True
             )
-        self.verify_disks_lso_attached()
-        self.do_click(self.dep_loc["next"], enable_screenshot=True)
+        if config.ENV_DATA.get("platform") != constants.BAREMETAL_PLATFORM:
+            self.verify_disks_lso_attached()
+            timeout_next = 60
+        else:
+            timeout_next = 600
```

Comment:
this is a significant timeout. I suggest we poll the status of a certain resource that can indicate whether or not we can proceed to the next step

Reply:
Can you maybe elaborate why this is 10times more for BM deployment to go to next button?
Is it because it's not just clickable before and waiting till it's clickable or something like that?
Can we maybe wrap the line below with some retry on the specific exception with some e.g 30 seconds in between attempts to retry?

Reply:
On BM we skip on `verify_disks_lso_attached` method because the disk size is not clear, and we can replace the disk on BM. [on LSO we add the disk "manually" and we can control the disk size. So my solution is waiting more time until we can click on the "next" button.

Reply:
https://url.corp.redhat.com/70d7466

Reply:
https://github.com/red-hat-storage/ocs-ci/pull/9415/files#diff-99d9a70366ab4f661fc2d565da8912b6664152141c29c0c6b5ac2da39c9d1c59L48

Reply:
Talked to Oded, and he explained me that the timeout is just timeout to wait maximum time for element to be clickable. Which means if it will be clickable in 30 seconds, it will continue in 30 seconds, not 600, but if the element is not clickable in 600 seconds (maximum timeout) it will fail the deployment.